app/controllers/ws_connection_controller.rb

```diff
@@ -0,0 +1,26 @@
+# Manage the connections
+class WsConnectionController < WebsocketRails::BaseController
+  def initialize_session
+    # perform application setup here
+    controller_store[:message_count] = 0
+  end
+
+  def client_connected
+    return if current_driver.nil?
+    WebsocketRails.users[current_driver.id] = connection
+    @orders = Order.where(status: %w(waiting arrived accepted)) \
+                   .where(driver_id: current_driver.id)
+    if @orders.length == 0
```

Comment:
```
$ rubocop app/controllers/ws_connection_controller.rb 
Inspecting 1 file
C

Offenses:

app/controllers/ws_connection_controller.rb:8:3: C: Assignment Branch Condition size for client_connected is too high. [16.61/15]
  def client_connected
  ^^^
app/controllers/ws_connection_controller.rb:13:5: C: Use the return of the conditional for variable assignment and comparison.
    if @orders.length == 0 ...
    ^^^^^^^^^^^^^^^^^^^^^^
app/controllers/ws_connection_controller.rb:13:8: C: Use @orders.length.zero? instead of @orders.length == 0.
    if @orders.length == 0
       ^^^^^^^^^^^^^^^^^^^
app/controllers/ws_connection_controller.rb:13:8: C: Use empty? instead of length == 0.
    if @orders.length == 0
       ^^^^^^^^^^^^^^^^^^^

1 file inspected, 4 offenses detected
```